lib/daru.rb

```diff
@@ -58,11 +61,21 @@ def create_has_library(library)
         class_variable_get(cv)
       end
     end
+
+    def plotting_library= lib
+      case lib
+      when :gruff, :nyaplot
+        @plotting_library = lib
+      else
+        raise ArgumentError, "Unsupported library #{lib}"
+      end
+    end
```

Comment:
Hmmm yes you have a point. The former will make it more extensible and probably more usable. I think we can have a further look at this when we make a uniform plotting interface.

Is your main concern the `plotting_library` variable in DF/Vector since it's pulling in plotting functionality into core daru data structures?